tests/Service/HelloAssoClientTest.php

```
@@ -5,39 +5,165 @@
             $this->httpClient,
-            $this->logger
+            $this->logger,
+            $this->cache,
         );
     }
 
-    public function testLoginReturnsAccessToken()
+    private function mockCachePassthrough(): void
     {
+        $this->cache->method('get')->willReturnCallback(
+            fn (string $key, callable $callback) => $callback($this->createMock(ItemInterface::class))
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$key'. (undefined)

(UnusedFormalParameter)